js/ui/form/ui.form.layout_manager.js

```diff
@@ -350,8 +350,28 @@ var LayoutManager = Widget.inherit({
         this._refresh();
     },
 
+    _renderTemplate: function($container, item) {
+        switch(item.itemType) {
+            case "empty":
+                this._renderEmptyItem($container);
+                break;
+            case "button":
+                this._renderButtonItem(item, $container);
+                break;
+            default:
+                this._renderFieldItem(item, $container);
+        }
+    },
+
+    _renderTemplates: function(templatesInfo) {
+        templatesInfo.forEach((function(info) {
```

Comment:
why not use [each](https://github.com/DevExpress/DevExtreme/blob/18_1/js/ui/form/ui.form.layout_manager.js#L14) utility?